src/language/__tests__/blockString-test.js

```diff
@@ -174,4 +174,15 @@ describe('printBlockString', () => {
       ),
     );
   });
+
+  it('correctly prints single-line with trailing backslash', () => {
+    const str = 'backslash \\';
+
+    expect(printBlockString(str)).to.equal(
+      joinLines('"""', 'backslash \\', '"""'),
+    );
+    expect(printBlockString(str, '', true)).to.equal(
+      joinLines('"""', 'backslash \\', '"""'),
+    );
+  });
```

Comment:
It makes more sense to move it one test up since has the same logic.
Also `joinLines` make sense only in tests related to "indent".

Reply:
done.